src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -769,29 +772,180 @@ public override string ToString()
     }
 }
 
+struct SizeSlot
+{
+    public uint size;
+    public uint count;
+    SizeSlot(uint size, uint count)
+    {
+        this.size = size;
+        this.count = count;
+    }
+
+    // an object size distribution for SOH/LOH derived from a real server scenario
+    public static SizeSlot[] sohLohSizeSlots = new SizeSlot[]
+    {
+//        new SizeSlot(        24,  16229766), We are creating some small wrapper objects of 32 and 40 bytes around the
+//        new SizeSlot(        32,  48790439), payload objects - somewhat compensate for this by distorting the distribution
+//        new SizeSlot(        40,  21496250), at the low end
+//        new SizeSlot(        48,  15020039),
+//
+        new SizeSlot(        56,  12134890),
```

Comment:
NIT but maybe for later having the ability to read these distributions from a text file might be helpful in case we want to allow the users to specify these.

Reply:
In a sense we already have that capability, because the text file given with "-file" can contain definitions of size buckets. But for many buckets, it's inconvenient because you'd have to specify survivorships and other properties for each bucket. Let's think about which capabilities we really want here...